tests/rewards/test_reward_nets.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import numbers
 import os
+from tempfile import TemporaryDirectory
```

Comment:
Import modules not individual classes: https://google.github.io/styleguide/pyguide.html#22-imports